client/web/src/repo/blob/own/FileOwnershipEntry.tsx

```diff
@@ -23,7 +23,19 @@ export const FileOwnershipEntry: React.FunctionComponent<Props> = ({ owner, reas
         setIsExpanded(!isExpanded)
     }, [isExpanded])
 
-    const email = owner.__typename === 'Person' ? owner.email : undefined
+    const findEmail = (): string | undefined => {
+        if (owner.__typename !== 'Person') {
+            return undefined
+        }
+        const primaryEmail: { __typename?: 'UserEmail' | undefined; email: string } | null | undefined =
+            owner?.user?.primaryEmail
+        if (primaryEmail?.__typename === 'UserEmail') {
+            return primaryEmail.email
+        }
+        return owner.email
```

Comment:
found this a bit hard to parse, wdyt about

```suggestion
        if (owner.user && owner.user.primaryEmail !== null) {
            return owner.user.primaryEmail.email
        }
        return owner.email
```

Reply:
Yes! Apologies, this is my TypeScript confusion.

With some elvises it parses.